app.rb

```diff
@@ -220,26 +220,23 @@ class InvalidChannelName < StandardError; end
     JSON.pretty_generate(available_versions.last)
   end
 
-  get /(?<channel>\/[\w]+)?\/(?<project>[\w-]+)\/platforms\/?$/ do
+  get /(?<channel>\/[\w]+)?\/(?<project>[\w-]+)\/platforms/ do
     pass unless project_allowed(project)
+    redirect '/platforms', 302
+  end
 
-    # Historically this endpoint was being used by chef-web-downloads to
-    # extend the platform keys with a human friendly versions.
-    # Now that this functionality is not needed anymore we can deprecate this
-    # endpoint. However in order not to break others who might be calling into
-    # this we return our full set of platform key to user friendly platform
-    # name mappings.
+  get /platforms/ do
     JSON.pretty_generate({
       "aix"       => "AIX",
       "el"        => "Enterprise Linux",
```

Comment:
Should we elaborate on what Enterprise Linux actually means. This was something that really confused me when I first started using Chef.